apps/api/migrations/082_add_savings_schedules_onchain_mandate_id.up.sql

```
@@ -0,0 +1,13 @@
+-- Add on-chain mandate ID to savings schedules for issue #808
+-- This links off-chain savings schedules with on-chain recurring deposit mandates
+
+ALTER TABLE savings_schedules 
+ADD COLUMN onchain_mandate_id BIGINT NULL;
+
+-- Create index for efficient lookups by mandate ID
+CREATE INDEX IF NOT EXISTS idx_savings_schedules_onchain_mandate_id 
+ON savings_schedules(onchain_mandate_id) 
+WHERE onchain_mandate_id IS NOT NULL;
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)